gokart/task.py

```diff
@@ -178,6 +179,10 @@ def dump(self, obj, target: Union[None, str, TargetOnKart] = None) -> None:
         self._get_output_target(target).dump(obj)
 
     def make_unique_id(self):
+        self.task_unique_id = self.task_unique_id or self._make_unique_id()
+        return self.task_unique_id
+
+    def _make_unique_id(self):
```

Comment:
I think this naming is bad because it's different from the actual condition of `make_unique_id` .
Maybe, there is a good naming, for example `_make_hash` , `_make_hash_id` ,.... How do you think? @liyuan9988

Reply:
I see. I would prefer `_make_hash_id`.
How about `self.task_unique_id`? Should we change it to `self.task_hash_id`?

Reply:
I think `task_unique_id` method does not need to be changed.

I was worried about having `task_unique_id` and `_task_unique_id`. The functions were completely different though they had similar names.

Reply:
fixed